de-lib/src/main/java/org/iplantc/de/theme/base/client/analyses/AnalysesMessages.properties

```diff
@@ -38,3 +38,14 @@ shareSupportMi = Share with support...
 shareSupportConfirm = Your output folder(s) and file(s) will be shared with CyVerse support team. Do you want to continue ?
 shareWithInput = Yes. Share my input files as well!
 shareOutputOnly = Share output folder(s) only!
+outputUnexpected = Analysis completed but I got unexpected output.
+noOutput = Analysis completed but I got no output.
+selectCondition = Select the condition of your analysis:
+comments = Comments
+needHelp = I still need help!
+submit = Submit
+agreeToShare = <span style="overflow:wrap;">I agree to <a href="https://wiki.cyverse.org/wiki/x/KBfHAQ" target="_blank">share my analysis</a> details, input file(s), output file(s) and logs with CyVerse Support. <br/>If you want any of the <a href="#" target="_blank">Science Informaticians</a> to recuse themselves from assisting with this problem please list their name(s) in comments.</span>
```

Comment:
Is the phrase `Science Informaticians` supposed to be a link? Also, there should probably be a comma in `with this problem, please list`.